app/routes/index.js

```diff
@@ -0,0 +1,10 @@
+import Route from "@ember/routing/route";
+import { inject as service } from "@ember/service";
+
+export default class IndexRoute extends Route {
+  @service session;
+
+  beforeModel() {
+    this.session.requireAuthentication(null, "auth");
```

Comment:
```suggestion
  beforeModel(transition) {
    this.session.requireAuthentication(transition, "auth");
```

`requireAuthentication` needs the transition so it can retry that once the session has been authenticated.